sklearn/tests/test_multiclass.py

```diff
@@ -314,14 +314,17 @@ def test_ovr_multilabel_predict_proba():
         X_test = X[80:]
         clf = OneVsRestClassifier(base_clf).fit(X_train, Y_train)
 
-        # decision function only estimator. Fails in current implementation.
+        # Decision function only estimator.
         decision_only = OneVsRestClassifier(svm.SVR()).fit(X_train, Y_train)
-        assert_raises(AttributeError, decision_only.predict_proba, X_test)
+        assert not hasattr(decision_only, 'predict_proba')
```

Comment:
these are pytest style asserts, while we are still using nosetests. Maybe use "assert_true" and "assert_false" until we transition?